src/Numerical/HBLAS/BLAS/Internal.hs

```diff
@@ -182,6 +186,35 @@ gemvAbstraction gemvName gemvSafeFFI gemvUnsafeFFI constHandler = gemv
                          (fromIntegral bstride) betaPtr cp (fromIntegral cstride)
 
 
+{-# NOINLINE gerAbstraction #-}
+gerAbstraction :: (SM.Storable el, PrimMonad m)
+               => String
+               -> GerFunFFI el
+               -> GerFunFFI el
+               -> forall orient . GerFun el orient (PrimState m) m
+gerAbstraction gerName gerSafeFFI gerUnsafeFFI = ger
+    where
+      shouldCallFast :: Int -> Int -> Bool
+      shouldCallFast m n = flopsThreshold >= (fromIntegral m :: Int64)
```

Comment:
Yes, I think we should be calling the fast code IF m*n is *LESS*(or *LEQ*) than flopsThreshold. But I think it is the same that we should be calling the fast code IF flopsThreshold is *GREATER*(or *GEQ*) than m*n?